recipe/0309-annotation-collection/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Using Annotation Collections
+id: 309
+layout: recipe
+tags: [tbc]
+summary: "tbc"
+viewers:
+topic: 
+ - basic
+---
+
+## Use Case
+
+You have a large number of Annotations covering multiple Canvases in a Manifest, for example crowd-sourced transcriptions or the output from a layout analysis algorithm and you want the Annotations to be displayed in a viewer as a recognizable group over the document with a label and a common color.
```

Comment:
Perhaps lead with one of the examples as the case and then the technical terms followed by the analogous use case examples.